### DISCUSSION TITLE
Move /etc/bluebanquise/pxe to /etc/bootset

### DISCUSSION THREAD

**Discussion by oxedions** [bluebanquise/bluebanquise#484] [Ideas]
Hello,

Having **bootset** parameters inside the `/etc/bluebanquise` folder is not a good design. `/etc/bluebanquise` should contains only static files, and generated files should be somewhere. It is also mandatory to ease multi iceberg usage.

I suggest we move all the pxe generated files to `/etc/bootset`.

Would that be ok for everyone ?

**Comment by hmeScaler** [comment]
Hello,
If it helps to have a good design.
I have no problem with that.
Regards

**Comment by oxedions** [comment]
Thanks a lot for the feedbacks.

Then I will do two things at the same time. bootset will be releaded in an rpm (like disklessset will be), and bootset params will be putt into /etc/bootset.
